frontend/src/pages/Login/Login.scss

```diff
@@ -0,0 +1,41 @@
+.Login {
+
+  header {
+    display: flex;
+    justify-content: space-between;
+    align-items: center;
+    padding: 30px;
+    box-shadow: var(--shadow-button);
+    margin-bottom: 30px;
+
+    h2 {
+      display: inline-block;
+      margin: 0;
+    }
+  }
+
+  form {
+    margin-left: 30px;
+
+    > * {
+      margin: 0px;
+      margin-bottom: 10px;
+    }
```

Comment:
Fancy stuff dude! Be careful with wildcards though, they can have unexpected consequences. Might be better to explicitly define the margins on the classes of each element, since some elements will naturally have larger margins to keep the visual hierarchy.

Reply:
I believe Kyle added that or it was inherited from another file. We will keep it for now but will be aware and update if another important change is made to the same file